### PR TITLE
Try It screen: improve handling of meta parameters

### PR DESCRIPTION
**Pull request by mauhiz**
- Do not send empty meta parameters by default
- Enforce required meta parameters

### CODE REVIEW

File: app/views/part/test.scala.html [hunk 1]
```diff
@@ -106,14 +108,17 @@
         <fieldset>
             <legend>@Messages("tryIt.meta")</legend>
             @for(metaProp <- metaProps) {
-                <div class="form-group">
-                    <label for="meta.@metaProp" class="col-sm-2 control-label">@metaProp</label>
-                    <div class="row">
-                        <div class="col-sm-5">
-                            <input name="meta.@metaProp" id="meta.@metaProp" type="text" class="form-control" />
+                @defining(part.registeredParamsView.find(_.inputNameJs == s"meta.$metaProp")) { metaParam =>
```
Comment (lloydmeta) [inline app/views/part/test.scala.html]: This changes reminds me: we should probably clarify somewhere:

1. What are meta parameters in the first place, how they get parsed and how they get passed.
2. If and when they should be declared

File: app/views/part/test.scala.html [hunk 2]
```diff
@@ -106,14 +108,17 @@
...
+                    <div class="form-group">
+                        <label for="meta.@metaProp" class="col-sm-2 control-label">@metaProp</label>
+                        <div class="row">
+                            <div class="col-sm-5">
+                                <input name="meta.@metaProp" id="meta.@metaProp" type="text" class="form-control @{metaParam.fold("optional")(_.requiredCls)}"/>
```
Comment (lloydmeta) [inline app/views/part/test.scala.html]: `requiredCls` is probably something that should be declared here in this view (it's only used here) as a defined as a local function or as helper inside the `view` package.

Right now, it's declared inside a presentation class and then we have to to funky stuff like `fold("optional")` which somewhat defeats the purpose of it. Having the CSS-related stuff (like class) in the template or somewhere close-by is more straightforward.

Comment (mauhiz) [inline app/views/part/test.scala.html]: which view? The fold(optional) means is there is no such registered parameter, make the meta param optional

Comment (lloydmeta) [inline app/views/part/test.scala.html]: Ah, by "view" I meant template. `requiredCls` declared on the class `ParamView` seems to be not used anywhere except in this template.

File: app/views/part/test.scala.html (hunk 1 above)
Comment (xevix) [inline app/views/part/test.scala.html]: :+1:  https://github.com/m3dev/octoparts-site/issues/35

File: app/views/part/test.scala.html (hunk 2 above)
Comment (mauhiz) [inline app/views/part/test.scala.html]: OK, I moved it there. Not sure how better it is, but I don't mind either way.

**Pull request merged**